src/errors/errorCodes.ts

```diff
@@ -8,7 +8,7 @@ export enum ApiErrorMessages {
   ERROR_SIGNATURE_CREATION = "Error on creating the JWS Signature",
   WALLET_OPTIONS_NOT_PROVIDED = "Wallet options parameters are not provided",
   BAD_CREDENTIAL_PARAMETERS = "Credential parameters are not valid.",
-  SIGNATURE_BAD_PARAMS = "Signature body requires issuer, payload and type.",
+  SIGNATURE_BAD_PARAMS = "Signature body requires did, issuer and payload.",
```

Comment:
issuer, payload and password

Reply:
> Regarding the openapi, a minor change I would suggest is that all the summaries and their despcriptions such as:
> 
> * Validates a W3C Verifiable Credential eSealed with an EIDAS QEC
> * Set up a DID with an eIDAS QEC to eSeal W3C Verifiable Credentials
> * ...
> 
> All of them use either the infinitve or the 3rd person.

You mean to change the descriptions to be ALL either infinitive of 3rd person? One case or the other?

Reply:
Yes. I meant that.